Makefile

```diff
@@ -407,17 +407,28 @@ ifeq ($(USE_KIND_CLUSTER),true)
 #	* Some KUTTL versions are not able to start properly a Kind cluster
 #	* The cluster will be removed after running KUTTL (this can be disabled). Sometimes,
 #		the cluster teardown is not done properly and KUTTL can not be run with the --start-kind flag
-# When the Kind cluster is not created by Kuttl, the
-# kindContainers parameter from kuttl-tests.yaml has not effect so, it is needed to load the
-# container images here.
+# When the Kind cluster is not created by Kuttl, the kindContainers parameter
+# from kuttl-tests.yaml has not effect so, it is needed to load the container
+# images here.
 	$(VECHO)$(KIND) create cluster --config $(KIND_CONFIG) 2>&1 | grep -v "already exists" || true
+# Install metrics-server for HPA
+	$(ECHO)"Installing the metrics-server in the kind cluster"
+	$(VECHO)kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.6.1/components.yaml
```

Comment:
In terms of consistency, does it make sense to move that link/version up like other external links?

https://github.com/jaegertracing/jaeger-operator/blob/2ccf2d4a4ab799ba07a4c408bde8a2caad3d16f8/Makefile#L35-L46

Reply:
Good idea!